build.sbt

```diff
@@ -579,6 +595,22 @@ lazy val commonSettings = Def.settings(
   })
 )
 
+lazy val enableMimaSettingsJVM =
+  Def.settings(
+    mimaPreviousArtifacts := previousStableVersion.value.map(organization.value %% moduleName.value % _).toSet,
+    mimaBinaryIssueFilters ++= Seq( // Filters should be cleared after a new release
+      ProblemFilters.exclude[MissingTypesProblem]("caliban.schema.SchemaDerivation$Auto"),
+      ProblemFilters.exclude[ReversedMissingMethodProblem](
+        "caliban.schema.SchemaDerivation#Auto.caliban$schema$SchemaDerivation$Auto$$$outer"
+      )
```

Comment:
The changes to `Schema.Auto` are not bincompat but that's okay since it wasn't working as intended previously